README.md

```diff
@@ -2,14 +2,19 @@
 Utilities for evaluating large genetic genealogies (e.g. ARGs) stored in the [tskit](https://tskit.dev) "succinct tree sequence" format.
 This is targetted at ARGs that have been inferred using a variety of inference tools such as `tsinfer`, `sc2ts`, `Relate`, `kwARG`, `ARGneedle`, etc. 
 
+To view a tskit tree sequence or tszip file first pre-process it:
+
+`python -m tsbrowse preprocess /path/to/trees-file`
+
+This will write a `.tsbrowse` file
 
 To launch the app use:
 
-`python -m tsbrowse /path/to/trees-file`
+`python -m tsbrowse serve/path/to/tsbrowse-file`
```

Comment:
I guess we're missing a space between `serve` and `/path/` here

Reply:
Good spot!